### DISCUSSION TITLE
How do (or can) I play osu!lazer on a x32 system?

### DISCUSSION THREAD

**Discussion by klevergleb** [ppy/osu#19117] [Q&A] [answered]
Hello, dear osu! players. So, recently I learned about osu!lazer and ofc tried to play it instead of normal osu! (it runs perfectly on my system), but when I launched the installer, it told me that I'll need an x64 system to play it, even tho peppy himself said that lazer will have equal or lower than stable in his [lazer FAQ](https://blog.ppy.sh/a-definitive-lazer-faq/). Why and is there a some sort of workaround for this?

**Comment by peppy** [comment, marked as answer]
What are your system specs? Specifically, what operating system are you running?

**Reply by klevergleb** [reply]
So, I'm running Windows 7 Ultimate x32 on my main PC, it has AMD Athlon X2 64 as CPU and 4 gigs of RAM.

**Reply by peppy** [reply]
Windows 7 is not supported by lazer, so that will still be a blocker. Please use stable.

**Reply by klevergleb** [reply]
That's sad, but why?

**Reply by smoogipoo** [reply]
Because our database dependency doesn't support it.
